config/wdio/clinical-lowlight-theme/wdio.conf.js

```diff
@@ -0,0 +1,15 @@
+const defaultConfig = require('../../../node_modules/terra-toolkit/config/wdio/wdio.conf');
+
+const config = {
+  ...defaultConfig.config,
+  axe: {
+    ...defaultConfig.config.axe,
+    options: {
+      rules: [
+        { id: 'color-contrast', enabled: false },
```

Comment:
Is there an issue logged to investigate and fix all accessibility issues related to color contrast?

Reply:
As per an offline conversation with @neilpfeiffer, the color contrast items may or maynot be addressed for lowlight (still undecided). Since, the color contrast ratios and rules are designed specifically for normal light situations, not specialized lighting situations like how the lowlight situations are used (radiology, etc).

Reply:
👍 Let's add an inline comment for future reference. To me, without background on the theme, these seems like something that should be fix so it would be nice to know in the future this was intentional and doesn't need cleaned up.

Reply:
Added an inline comment : https://github.com/cerner/terra-core/pull/2758/commits/f0b4184ca3d941e38b02ceb6ee8fb5f386d9efd2